waiter/project.clj

```diff
@@ -85,6 +85,7 @@
                   :exclusions [org.clojure/tools.reader]]
                  [ring-basic-authentication "1.0.5"]
                  [slingshot "0.12.2"]
+                 [swiss-arrows "1.0.0"]
```

Comment:
Where is this used? Do we really need it?

Reply:
I was using `-<>` and `-<>>` in an earlier version of this feature, but it looks like I completely eliminated those when I was refactoring. I removed this dependence. Thanks for catching that.

Reply:
I added this dependency back again so I could use `swiss.arrows/<<-` in the unit test code.

Reply:
I would vote for avoiding importing `swiss.arrows` for the unit tests.

Reply:
I should be able to move this so it's only pulled in when running unit tests. If it's not used in the non-test code, then it doesn't need to be in this dependency list.

Reply:
I'll delete this and use `_ (do ...)` in the let bindings list instead of having the 10+ successive let blocks.